modules/ROOT/pages/http-caching-policy.adoc

```diff
@@ -6,241 +6,240 @@ endif::[]
 
-The response isn’t stored. For it to be stored or searched in the cache, all conditions must be positive, if at least one condition says the cache shouldn’t intervene, it won’t.
 
-=== Can I store only part of the response with this policy?
+*Can I store only part of the response with this policy?*
 
 No, but you can apply a policy before this one and transform the response to something else.
 
-=== What behaviors are different in Runtime Manager?
+*What happens if one parameter is configured to search for the response in the cache and the other parameter is configured for the opposite action?*
 
-* If the distributed checkbox is checked, it's always persistent.
-* The maximum cache entry doesn’t apply in Runtime Manager.
-* Edition of the TTL of the policy won't be correctly reflected while using object store V2; to change this value remove the policy and apply a new one.
+This is the same as configuring the response to be stored in the cache when there is a no-store directive in the Cache-Control header.
 
-=== Do I need to configure something to use the policy in Runtime Manager?
+In this situation, the response isn’t stored. You must configure both the Cache-Control header and the cache to be able to store and search for the response in the cache.
 
-Yes, if you want for the cache to be distributed or persistent, you have to set the Object Store as v2 in the Runtime Manager.
 
-=== Which data types are supported to store in the cache?
+*How does running the HTTP Caching policy in CloudHub differ compared with running it in Mule runtime engine?*
 
-Java serializables and input streams can be cached. In the case of input streams, they are read up to the maximum size of 1 MB that a cache entry can have. If that value is surpassed, then reading stops and the store is avoided.
+* If the Distributed option is selected, the cache is always persistent.
+* The maximum cache configuration is ignored in Runtime Manager.
+* The edition of the TTL of the policy is not correctly reflected while using Object Store v2. To change this value, remove the HTTP Caching policy and apply a new one.
 
-=== What happens if I don’t define some of the optional parameters?
 
-* If invalidation header is not set, it won’t be possible to invalidate the cache in a request.
-* If the request expression or response expression are not set, the cache is used for all requests, and stores all the responses in the cache respectively, as if the expression `#[true]` was set.
+*What happens if I don’t define some of the optional parameters?*
 
-=== Can I change the Invalidation Header values?
+If you have not configured the Invalidation header, you cannot invalidate the cache in a request.
+Additionally, if you have not configured the request expression or response expression, the cache is used for all requests. All the responses are stored in the cache respectively (expression #[true]).
```

Comment:
What does "respectively" mean here? We typically use it to mean "in the same order" or "to both things in the same order." So, the cache is used only for requests, or for all the requests, first, and then all the responses thereafter?

Reply:
Done! Thanks!